modules/ROOT/pages/salesforce/salesforce-composite-connector.adoc

```diff
@@ -291,6 +291,197 @@ An example of HTTP input for this flow is:
 }
 ----
 
+== Pre-query Example
+
+This example provides metadata for an object to be queried, generates two sub-requests that contain this metadata, and then executes the sub-requests. The following screenshots show the flow for the Pre-Query example:
+
+.The flow for this example includes an HTTP Listener component, Transform components, and multiple Salesfore Composite operations.
+image::salesforce/salesforce-composite-prequery-flow.png[Pre-Query flow in Studio]
+
+=== Start Creating the Flow by Adding HTTP Listener
+
+`HTTP Listener` initiates the flow when it detects an event in the `executeFlow` path:
+
+. Create a new Mule project in Studio.
+. In the Mule Palette view, search for *HTTP* and select the *Listener* operation.
```

Comment:
GLOBAL: "Mule Palette" appears as a label in the Studion interface, if I remember correctly, so it should be formatted in bold:
```suggestion
. In the *Mule Palette* view, search for *HTTP* and select the *Listener* operation.
```